diff --git a/debian/python3-onedatafs-jupyter.install b/debian/python3-onedatafs-jupyter.install
index 8949d75..452be65 100644
--- a/debian/python3-onedatafs-jupyter.install
+++ b/debian/python3-onedatafs-jupyter.install
@@ -1,1 +1,1 @@
-.pybuild/pythonX.Y_3.5/build/onedatafs_jupyter/* usr/lib/python3.5/dist-packages/onedatafs_jupyter
+.pybuild/cpython3_3.8_onedatafs-jupyter/build/onedatafs_jupyter/* usr/lib/python3.8/dist-packages/onedatafs_jupyter
